R/init.R

```diff
@@ -41,6 +55,8 @@ init_rstudio <- function(
   rhino_version = "rhino"
 ) {
   init_impl(
+    # No need to check if dir is home, because RStudio's new project wizard already requires that
+    # the user provide a directory within the home directory.
```

Comment:
You can actually create a project outside of your home directory :wink:
```suggestion
    # No need to check if `dir` is home,
    # because RStudio's new project wizard always creates a new directory.
```